flaskr/blog.py

```diff
@@ -22,6 +23,8 @@ def index():
         " FROM post p JOIN user u ON p.author_id = u.id"
         " ORDER BY created DESC"
     ).fetchall()
+    for post in posts:
```

Comment:
posts is immutable, this is not going to work

Reply:
I've pushed the changes as required.